packages/uikit-workshop/src/scripts/components/pl-nav/src/NavList.js

```diff
@@ -0,0 +1,68 @@
+import { h } from 'preact';
+import { NavToggle } from './NavToggle';
+import { NavLink } from './NavLink';
+import { NavItem } from './NavItem';
+
+export const NavList = props => {
+  const { children, category, elem } = props;
+  const reorderedChildren = [];
+
+  const nonViewAllItems = elem.noViewAll
+    ? children.filter(item => item.patternName !== 'View All')
+    : children.filter(
+        item =>
+          item.patternName !== 'View All' && !item.patternName.includes(' Docs')
+      );
```

Comment:
Im not shure where patterns with `Docs` come from, I cant find anything in the core package regarding this pattern name.

Reply:
Ah. This was something I started to play around with (upstream from Pattern Lab) in the [Bolt Design System](https://boltdesignsystem.com/pattern-lab/?p=viewall-components-progress-bar). 

Basically it's a yet to be documented feature that lets you replace the generic "View All" template with something a bit more custom (ie. extra info like the component name, description, status, etc).